.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
```

Comment:
If I have Error in both lines and click Enter on keyboard or = character then it shows me the previous result (I think even calculated in the lower line), it shouldn't, C operation is needed to clear errors and new operations must be done, we don't operate on errored operations
![image](https://github.com/user-attachments/assets/df81555e-cf96-4af2-8543-f602d0e6fe77)